site/README.md

```diff
@@ -12,6 +12,7 @@ The top bar is customized with the sections I think make sense to split. However
 ## Building
 
 Build and run using Docker or Hugo, then access the site at `http://localhost:1313`.
+Please note that this directory contains nested submodules. Follow instructions in [the git submodules documentation](https://git-scm.com/book/en/v2/Git-Tools-Submodules) to initialize.
 
```

Comment:
How about actually suggesting the commands we should run? 
There are many no-code contributors in the open source community, who is not familiar with git, so I think it wouldn't hurt to be specific.

```bash
git clone https://github.com/kubernetes-sigs/kustomize
cd kustomize
git submodule update --init --recursive
cd site
```

Reply:
Done.

Really great point, @bugoverdose! I had to run the commands again to add this part of the documentation, proving it wasn't trivial. Really appreciate you highlighting this oversight!